src/container/srv_layout.h

```diff
@@ -71,17 +71,20 @@ extern d_iov_t ds_cont_prop_cont_handles;	/* container handle KVS */
  */
 extern d_iov_t ds_cont_prop_ghce;		/* uint64_t */
 extern d_iov_t ds_cont_prop_max_oid;		/* uint64_t */
-extern d_iov_t ds_cont_prop_label;		/* uint64_t */
+extern d_iov_t ds_cont_prop_label;		/* string */
```

Comment:
I'm assuming this is just from a merge from master? Or are we changing the label?

Reply:
In this case it just looked like the comment was incorrect due to copy/paste errors. Label is a string.